kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -39,6 +69,22 @@ def airflow_commands():
         exists=True, readable=True, resolve_path=True, file_okay=True, dir_okay=False
     ),
     default=Path(__file__).parent / "airflow_dag_template.j2",
+    help="The template file for the generated Airflow dags",
+)
+@click.option(
+    "--params",
+    type=click.UNPROCESSED,
+    default="",
+    help=PARAMS_ARG_HELP,
+    callback=_split_params,
+)
+@click.option(
```

Comment:
Instead of having this as a CLI flag, might be good to have users add this to their `settings.py` as one of the `config_patterns`. Would like to hear what other people also think about this @noklam @astrojuanlu 
```
CONFIG_LOADER_ARGS = {
      "config_patterns": {
          "airflow" : ["airflow*", "airflow/**"],
      }
}
```

Reply:
Agreed, will update